File: JavaScript/4/4_4.js

const PRODUCT_NAME = "Widget"
const QUANTITY = 5

function updateInventory(product, quantity) {
  console.log(quantity + " units of " + product + " sold.");
  return inventory[product] - quantity
}
inventory[PRODUCT_NAME] = updateInventory("Widget", 5);
// idk

File: C#/4/4_4.cs
 Dictionary<string, int> inventory = new Dictionary<string, int>
{
    { "Widget", 10 }
};

void UpdateInventory(string product, int quantity)
{
    product -= quantity;
    Console.WriteLine(quantity + " units of " + product + " sold.");
}

UpdateInventory(inventory["Widget"], 5);
